library-management/src/app/components/edit-admin/edit-admin.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+
+import { Admin } from 'src/app/models/admin';
+import { AdminService } from 'src/app/services/admin.service';
+import { ToasterService } from 'src/app/services/toaster.service';
+
+@Component({
+  selector: 'app-edit-admin',
+  templateUrl: './edit-admin.component.html',
+  styleUrls: ['./edit-admin.component.css'],
+})
+export class EditAdminComponent implements OnInit {
+  editAdminForm: FormGroup;
+  admin: Observable<Admin> | any;
```

Comment:
dont use observable as datatype cuz we are not using observable to return and don't use  ** type|any **